src/bar/BarChartWithFilters.vue

```diff
@@ -135,13 +137,24 @@
         }
     };
 
-    export default Vue.extend<{}, Methods, Computed, Props>({
+    interface Data{
+        disaggregateIsFixed: boolean | null
+        xAsisIsFixed: boolean | null
+    }
+
+    export default Vue.extend<Data, Methods, Computed, Props>({
         name: "BarChart",
         props: props,
         model: {
             prop: "selections",
             event: "change"
         },
+        data(): Data {
+            return {
+                disaggregateIsFixed: this.disaggregateByConfig && this.disaggregateByConfig.fixed,
+                xAsisIsFixed: this.xAxisConfig && this.xAxisConfig.fixed,
```

Comment:
These should be in `computed` rather than `data`?
And, since you're adding these anyway, you could also use them in the template to simplify lines 13 and 20.

Reply:
Props can't be used in computed without setting a default value. I have reused values as suggested.

Reply:
Should be ok for those to default to null I think.